tests/phpunit/tests/link/getEditCommentLink.php

```diff
@@ -0,0 +1,107 @@
+<?php
+/**
+ * @group link
+ * @group comment
+ * @covers ::get_next_comments_link
+ */
+class Tests_Link_GetEditCommentLink extends WP_UnitTestCase {
+
+	public static $comment_ids;
+	public static $user_ids;
+
+	public static function wpSetUpBeforeClass( WP_UnitTest_Factory $factory ) {
+		self::$comment_ids = $factory->comment->create( array( 'comment_content' => 'Test comment' ) );
+
+		self::$user_ids = array(
+			'admin'      => $factory->user->create( array( 'role' => 'administrator' ) ),
+			'subscriber' => $factory->user->create( array( 'role' => 'subscriber' ) ),
+		);
+	}
+
+	public static function wpTearDownAfterClass() {
+		// Delete the test comment
+		if ( isset( self::$comment_ids ) ) {
+			wp_delete_comment( self::$comment_ids, true );
+		}
+
+		// Delete the test users
+		if ( isset( self::$user_ids['admin'] ) ) {
+			wp_delete_user( self::$user_ids['admin'] );
+		}
+
+		if ( isset( self::$user_ids['subscriber'] ) ) {
+			wp_delete_user( self::$user_ids['subscriber'] );
+		}
```

Comment:
No need to check for `isset()`, since those will always be set due to how the methods are called. We can simplify this:
```suggestion
		// Delete the test comment.
		wp_delete_comment( self::$comment_ids, true );

		// Delete the test users.
		foreach ( self::$user_ids as $user_id ) {
			self::delete_user( $user_id );
		}
```

Note that for multisite we need to use `wpmu_delete_user()` instead of `wp_delete_user()`. The WordPress test class has a static `delete_user( $user_id )` method that takes care of that.